src/app/RecommendationsList/removeButton.js

```diff
@@ -0,0 +1,20 @@
+import { GenerateRecomendationsSourceList } from "./recomentadionsSource";
+export class RemoveButton {
+    static lookForBtnUse() {
+        let removeBtn = document.querySelectorAll(".removeElement-button")
+        for (let i = 0; i < removeBtn.length; i++) {
+            removeBtn[i].addEventListener('click', () => {
+                const removeBtnParent = removeBtn[i].parentElement.parentElement
+                removeBtnParent.className += ' removeTransformation';
```

Comment:
classes, selectors, all extracted to consts, there should be no magic string in js ever